documentation/book/topic-operator.adoc

```diff
@@ -1,260 +1,55 @@
 == Topic Operator
 
-The Topic Operator is in charge of managing topics in a Kafka cluster. The Topic Operator is deployed as a process
-running inside a {ProductPlatformName} cluster.
-It can be deployed through the Cluster Operator or "manually" through provided YAML files.
-
-image::topic_operator.png[Topic Operator]
-
-The role of the Topic Operator is to keep a set of {ProductPlatformName} ConfigMaps describing Kafka topics in-sync with
-corresponding Kafka topics.
-
-Specifically:
- 
-* if a ConfigMap is created, the operator will create the topic it describes
-* if a ConfigMap is deleted, the operator will delete the topic it describes
-* if a ConfigMap is changed, the operator will update the topic it describes
-
-And also, in the other direction:
-
-* if a topic is created, the operator will create a ConfigMap describing it
-* if a topic is deleted, the operator will create the ConfigMap describing it
-* if a topic is changed, the operator will update the ConfigMap describing it
-
-This is beneficial to a {ProductPlatformName} centric style of deploying
-applications, because it allows you to declare a ConfigMap as part of your
-applications deployment and the operator will take care of creating
-the topic for you, so your application just needs to deal with producing 
-and/or consuming from the necessary topics.
-
-Should the topic be reconfigured, reassigned to different Kafka nodes and so on, 
-the ConfigMap will always be up to date.
+include::con_topic-operator.adoc[leveloffset=+2]
 
 
 === Reconciliation
```

Comment:
I think this section belongs more in design documentation rather than user doc. Or in a more detailed concept. Wdyt @scholzj @ppatierno ?